metrics/web/lib/project_groups/presentation/strings/project_groups_strings.dart

```diff
@@ -16,6 +16,7 @@ class ProjectGroupsStrings {
   static const String deletingProjectGroup = 'Deleting group...';
   static const String deleteConfirmation = 'Are you sure you want to delete';
   static const String deleteConfirmationQuestion = 'project group?';
+  static const String noSearchResults = 'no search results...';
```

Comment:
Should this string start from the capital letter according to designs? 
```suggestion
  static const String noSearchResults = 'No search results...';
```